src/runtime/rposix.r

```diff
@@ -1576,6 +1582,297 @@ int fd;
    MUTEX_UNLOCKID(MTX_SOCK_MAP);
 }
 
+
+#if HAVE_LIBSSL
+
+SSL_CTX * create_ssl_context(dptr attr, int n, int type ) {
+
+   SSL_CTX *ctx;
+   //      if (status & Fs_Encrypt) {
+   tended char *tmps, *val;
+   tended char *certFile=NULL, *keyFile=NULL, *ciphers=NULL, *ciphers13=NULL, *password=NULL;
+   tended char *ca_file=NULL, *ca_dir=NULL, *ca_store=NULL;
+   tended char *min_proto=NULL, *max_proto=NULL, *verifyPeer=NULL;
+   int old_ssl_flags=0;
+   static int SSL_is_initialized = 0;
+   C_integer timeout = 0, timeout_set = 0;
+   int count;
+   int a;
+
+   /* Check attributes */
+   for (a=0; a<n; a++) {
+     if (is:null(attr[a])) {
+       attr[a] = emptystr;
+     }
+     else if (a==0 && cnv:C_integer(attr[a], timeout)) {
+       timeout_set = 1;
+     }
+     else if (cnv:C_string(attr[a], tmps)) {
+
+       if (strlen(tmps) < 3 || tmps[0] == '=' || tmps[strlen(tmps)-1] == '=') {
+	 set_errortext_with_val(1217, tmps);
```

Comment:
Add 1217 to the list of errors in Appendix A.  (btw. I can't find a mention of 1216 anywhere -- the last error in the book is 1215).

Reply:
added the new errors and fix  the numbers